tests/test_draw.py

```diff
@@ -272,13 +272,32 @@ def test_draw():
     # Test if we provide nx, ny, and scale.  It should:
     #   - create a new image with the right size
     #   - set the scale
-    im10 = obj2.draw(nx=200, ny=100, scale=dx_nyq)
```

Comment:
This is a minor point, but I think it would be better to make the various instances of 200, 100, 201, 101, 0.51, etc. set by variables,
```
nx=200
ny=100
scale=0.51
```
and then refer to nx, ny, nx+1, ny+1, scale, etc.  That way if we have some reason to change this code in future, everything necessary will be defined up front.